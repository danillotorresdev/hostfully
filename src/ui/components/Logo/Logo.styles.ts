import { styled } from "@linaria/react";

export const Logo = styled.img`
  width: 200px;
  height: auto;
  display: block;
`;
