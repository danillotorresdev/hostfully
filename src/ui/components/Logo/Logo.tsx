import * as S from "./Logo.styles";

export const Logo = () => (
  <S.Logo
    src="https://www.hostfully.com/wp-content/uploads/2022/08/logo-1.svg"
    alt="Logo"
    style={{
      width: "200px",
      height: "auto",
      display: "block",
    }}
  />
);
