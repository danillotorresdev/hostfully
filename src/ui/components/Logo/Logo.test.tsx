import { describe, expect, it } from "vitest";
import { Logo } from "./Logo";
import { render, screen } from "@/utils/settings/tests/utilities";

describe("Logo component", () => {
  it("renders logo image with correct attributes", () => {
   const {asFragment} = render(<Logo />);

    expect(asFragment()).toMatchSnapshot();
    
    const logoImage = screen.getByAltText("Logo");
    expect(logoImage).toBeDefined();

    expect(logoImage).toBeDefined();
  });
});
