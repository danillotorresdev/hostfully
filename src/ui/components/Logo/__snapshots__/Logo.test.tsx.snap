// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Logo component > renders logo image with correct attributes 1`] = `
<DocumentFragment>
  <img
    alt="Logo"
    class="mocked-styled-0 le32cth"
    src="https://www.hostfully.com/wp-content/uploads/2022/08/logo-1.svg"
    style="width: 200px; height: auto; display: block;"
  />
</DocumentFragment>
`;
